cfdm/data/__init__.py

```diff
@@ -1,4 +1,7 @@
-from .abstract import Array, CompressedArray, MeshArray, RaggedArray
+from .abstract import Array
+from .abstract import CompressedArray
+from .abstract import MeshArray
+from .abstract import RaggedArray
```

Comment:
For my understanding, is there some (import speed or otherwise) benefit of importing individually in this way rather than as a one-liner?

Reply:
I don't know why these are on different lines - I'll put them back on one line